test/promql/tests/data_plane/gitops_sync_test.yaml

```diff
@@ -0,0 +1,101 @@
+evaluation_interval: 1m
+
+rule_files:
+  - prometheus.gitops_sync_alerts.yaml
+
+tests:
+  # Alerted cases:
+  - interval: 1m
+    input_series:
+
+      # GitOps service is experiencing application sync success rate of less than 95%, so alert
+      - series: 'argocd_app_info{sync_status="Synced",namespace="gitops-service-argocd"}'
+        values: '1 1 1 1 1 1'
+      - series: 'argocd_app_info{sync_status="OutOfSync",namespace="gitops-service-argocd"}'
+        values: '1 1 1 1 1 1'
+      - series: 'argocd_app_info{sync_status="Unknown",namespace="gitops-service-argocd"}'
+        values: '1 1 1 1 1 1'
```

Comment:
Those statuses should be mutually. In this case, the same app is in 3 different sync statuses at the same time.

Reply:
I added distinct names